brev-bestilling-boot/src/main/java/no/nav/pensjon/brev/bestilling/consumer/BrevMetaData.java

```diff
@@ -0,0 +1,46 @@
+package no.nav.pensjon.brev.bestilling.consumer;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.HttpStatusCodeException;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.util.UriComponentsBuilder;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+public class BrevMetaData {
+    private static final Logger LOG = LoggerFactory.getLogger(BrevMetaData.class);
+
+    @Value("${brevmetadata.endpoint.url}")
```

Comment:
Ikke bruk @Value på attribute nivår. Inject it via constructor.

Reply:
hvorfor? Er det ikke mye enklere å ha tomt constructor og styre attributer via injection? Når den hvil få flere attributer da constructor vil bli komplisert og da vil trenge som build pattern.

Reply:
Google constructor inject vs Attribute injection.